package.json

```diff
@@ -17,22 +17,22 @@
 	"author": "",
 	"license": "ISC",
 	"dependencies": {
-		"@hono/node-server": "^1.2.0",
+		"@hono/node-server": "^1.14.0",
 		"es-module-lexer": "^1.3.1",
-		"esbuild": "^0.17.14",
-		"hono": "^3.9.2",
-		"react": "18.3.0-next-3706edb81-20230308",
-		"react-dom": "18.3.0-next-3706edb81-20230308",
-		"react-server-dom-webpack": "0.0.0-experimental-41b4714f1-20230328"
+		"esbuild": "^0.25.1",
+		"hono": "^4.7.5",
+		"react": "^19.0.0",
+		"react-dom": "^19.0.0",
+		"react-server-dom-esm": "npm:@kentcdodds/tmp-react-server-dom-esm@19.0.1"
```

Comment:
curious why we're tying to a fork from kentcdodds here? We were previously on an experimental build, so I don't mind doing this. Just want to understand the choice.

Reply:
Alright, played around with some of the official APIs available, and there still doesn't seem to be an ESM version like this from the React Core team (sigh). I guess the Kent C Dodds fork is the way to go! I'll ask @kentcdodds about whether he still uses this fork as well.